### PR TITLE
feat: Add "Reply-to" to organizers in session notify

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes https://github.com/fossasia/open-event-frontend/issues/6544

new pr 2

### CODE REVIEW

File: app/api/helpers/mail.py [hunk 1]
```diff
@@ -173,6 +173,12 @@ def send_email_session_state_change(email, session, mail_override: Dict[str, str
     settings = get_settings()
     app_name = settings['app_name']
     frontend_url = settings['frontend_url']
+    organizersEmail = list(
```
Comment (iamareebjamal) [inline app/api/helpers/mail.py:176]: owner is missing

Comment (sachinchauhan2889) [reply app/api/helpers/mail.py:176]: ok sir i will update this.

File: app/api/helpers/mail.py [hunk 2]
```diff
@@ -179,6 +179,7 @@ def send_email_session_state_change(email, session, mail_override: Dict[str, str
             + list(map(lambda x: x.email, session.event.coorganizers))
         )
     )
+    organizersEmail.append(session.event.owner.email)
```
Comment (iamareebjamal) [inline app/api/helpers/mail.py:182]: Email may be duplicated

File: app/api/helpers/mail.py [hunk 3]
```diff
@@ -173,14 +173,9 @@ def send_email_session_state_change(email, session, mail_override: Dict[str, str
...
-        set(
-            list(map(lambda x: x.email, session.event.organizers))
-            + list(map(lambda x: x.email, session.event.coorganizers))
-        )
-    )
-    organizersEmail.append(session.event.owner.email)
-
+    organizers = list(map(lambda x: x.email, session.event.organizers)) + list(map(lambda x: x.email, session.event.coorganizers))
+    organizers.append(session.event.owner.email)
+    organizersEmail = list(set(organizers))
```
Comment (iamareebjamal) [inline app/api/helpers/mail.py:178]: snake case

**Pull request merged**